app/modules/incident/incident_helper.py

```diff
@@ -528,6 +514,20 @@ def schedule_incident_retro(client, body, ack):
                         "text": "How many days from now should I start checking the calendar for availability?",
                     },
                 },
+                {
+                    "type": "input",
+                    "block_id": "user_select_block",
+                    "label": {
+                        "type": "plain_text",
+                        "text": "Select everyone you want to include in the retro calendar invite",
```

Comment:
In this select user block, we have no users selected by default. Should we have an option to include all? Otherwise we'll have to manually pick everyone and may misclick or forget some users. I'm wondering if an "@everyone" or "@all" option should be available?

Reply:
Good idea! I will see if we can have an all checkbox. The reason why I did not select all by default is because the security team and Calvin always gets added to an incident but they usually never participate in retros, so I did not want to spam them. The problem with not including them is that if they want to participate in a retro, they couldn't before.

Reply:
OK, that makes sense! Definitely not a blocker I think so maybe we move ahead like this and see if we want to modify down the road 😄

Reply:
Created an issue for it and can be added as a feature later - https://github.com/cds-snc/sre-bot/issues/594
Unfortunately, given the API and how modals get refreshed, this is unfortunately harder to implement than I initially thought but can be added as a new feature later.